pages/linux/dnstracer.md

```diff
@@ -0,0 +1,28 @@
+# dnstracer
+
+> The dnstracer command determines where a DNS gets its information from.
+> More information: <http://www.mavetju.org/unix/dnstracer.php>.
```

Comment:
```suggestion
> More information: <https://manned.org/dnstracer>.
```

Here, we prefer using the `dnstracer`'s official [`man`](https://manned.org/man) documentation as hosted in manned.org. See #5143 for more info.

Reply:
Fine with me. I had linked the home page, because it was referred on the Kali tools page. `man` seems more sensible though.